app/main.py

```diff
@@ -1 +1,133 @@
-# write your code here
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```

Comment:
It returns int. You must specify type which you expect from all your child classes

Reply:
The same problem with method player_info